ee/dataflatten/string_delimited.go

```diff
@@ -15,6 +15,75 @@ const (
 	DuplicateKeys
 )
 
+// StringDelimitedLineFunc returns a function that conforms to the interface expected by
+// dataflattentable.Table's execDataFunc property. Properties are grouped into a single
+// record per line of data by splitting the line into fields based on a separator.
+//
+// Headers are expected to either be provided, or set from the first line of data using the separator.
+// If no separator is provided, it's assumed that fields are separated by whitespace.
+// The first N lines of data can be skipped in case garbage is sent before the data.
+func StringDelimitedLineFunc(sep string, headers []string, skipN int) dataFunc {
```

Comment:
I think it'd be nice to have unit tests for these new functions -- maybe in a follow-up PR later?

Reply:
For sure. I'll add that to my list of future things.